stl/src/ppltasks.cpp

```diff
@@ -19,7 +18,8 @@
 #include <ctxtcall.h>
 #include <mutex>
 #include <windows.foundation.diagnostics.h>
-#endif
+
+#pragma comment(lib, "ole32")
 
 // This IID is exported by ole32.dll; we cannot depend on ole32.dll on OneCore.

```

Comment:
No change requested: I see that your PR description says:

> Support for OneCore SKUs is assumed to be handled by ApiSet reverse-forwarding from the ole32.dll linkage.

Thus this comment is outdated. I can clean this up in a followup PR. (It looks like we still need the locally hardcoded GUID, as I can't seem to gain access to a valid declaration.)

Reply:
> No change requested: I see that your PR description says:
> 
> > Support for OneCore SKUs is assumed to be handled by ApiSet reverse-forwarding from the ole32.dll linkage.
> 
> Thus this comment is outdated. I can clean this up in a followup PR. (It looks like we still need the locally hardcoded GUID, as I can't seem to gain access to a valid declaration.)

Yes, comment can go, but local IID stays and is also defined across the Windows tree (and in cppwinrt base.h)